ShowdownAI.Middleware/ShowdownAI.Middleware/Models/TargetType.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ShowdownAI.Middleware.Models
+{
+    public enum TargetType
+    {
+        None,
+
+        // single-target
+        normal,
```

Comment:
In C# we normally write in CamelCase for things like enum values, class names, variables

Reply:
It was like that from the Showdown code. Would it matter if it is different? If not then I'll go ahead and change it.
![image](https://github.com/user-attachments/assets/d105c802-d0e0-4799-b206-09f304350738)

Reply:
Nah it won't make any difference. They've just written it in the Javascript style